tools/run_tests/run_interop_tests.py

```diff
@@ -1247,3 +1248,8 @@ def aggregate_http2_results(stdout):
       dockerjob.remove_image(image)
     else:
       print('Preserving docker image: %s' % image)
+
+if num_failures:
```

Comment:
It turns out finally does get executed even if I put sys.exit() inside the try block. So I'll move it.